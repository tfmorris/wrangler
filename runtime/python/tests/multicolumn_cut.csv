'hello','world'
'get','rid of'
'all of these','single quotes'
