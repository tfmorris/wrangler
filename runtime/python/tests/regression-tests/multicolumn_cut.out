split,split1
"hello","world"
"get","rid of"
"all of these","single quotes"
